src/main/java/com/faforever/client/preferences/LanguageInfo.java

```diff
@@ -0,0 +1,80 @@
+package com.faforever.client.preferences;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+public enum LanguageInfo {
+  AUTO("auto", null, "System", 0), EN("en", "UK", "English", 1), DE("de", "DE", "Deutsch", 2), RU("ru", "RU", "ру́сский", 3);
+
+  private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private final String languageCode;
+  private final String countryCode;
+  private final String dispayName;
+  private final int position;
```

Comment:
Enums implicitly have an `ordinal` which is their position in the definition above, so this can be removed.